src/tests/ftest/util/dfuse_utils.py

```diff
@@ -380,6 +381,20 @@ def stop(self):
         # Only assume clean if nothing above failed
         self.__need_cleanup = False
 
+    def get_stats(self):
+        """Return the I/O stats for the filesystem"""
+
+        cmd = f"daos filesystem query --json {self.mount_dir.value}"
+        result = run_remote(self.log, self.hosts, cmd)
+        if not result.passed:
+            raise CommandFailure(f'"{cmd}" failed on {result.failed_hosts}')
+
+        data = json.loads("\n".join(result.output[0].stdout))
+        print(data)
+        assert data["status"] == 0
+        assert data["error"] is None
+        return data["response"]
```

Comment:
This would only work on a single host, since it returns `output[0]`. Would different hosts potentially return different values? (then, I have further suggestions)

Reply:
Good point.

This is the dfuse I/O counters so yes, each host would have different values.  Do you mind if for now we just check that len(self.hosts) == 1?

Reply:
Yeah, that's the simplest solution for now and we can revisit if we need this for multiple hosts

Reply:
We should remove `print(data)` because (1) we should use `self.log.info` or `self.log.debug` instead, and (2) `run_remote` already prints all the output

Reply:
We shouldn't use direct `assert` here either. And in this case, I think we can raise a `CommandFailure` if the structure isn't correct